test/identify-test.es6

```diff
@@ -0,0 +1,51 @@
+
+import * as fs from 'fs';
+
+import {assert} from 'chai';
+import {identify} from '../lib/deckardcain';
+
+
+describe('API Blueprint', () => {
+  const path = `${__dirname}/apiblueprint-fixtures/`;
+
+  describe('with FORMAT header', () => {
+    const fixture = path + 'with-format-header.apib';
+    const source = fs.readFileSync(fixture, 'utf8');
+
+    it('is identified as API Blueprint', () => {
+      assert.equal(identify(source), 'text/vnd.apiblueprint');
+    });
+  });
+
+  describe('without FORMAT header but with any typical response', () => {
+    const fixture = path + 'with-typical-response.apib';
+    const source = fs.readFileSync(fixture, 'utf8');
```

Comment:
`readFileSync` is not crossplatform. I won't be able to run tests on the browser.
Could you require those fixtures with `import` statement?